Jenkinsfile

```diff
@@ -12,7 +12,14 @@ node(){
   }
 
   stage('Deploy')   {
-      cloudFoundryDeploy script:this, deployTool: 'mtaDeployPlugin'
+      def id = new Date().getTime()
+      consumerTestsSpace = "consr-test-${id}"
```

Comment:
As we only use one space per test execution we could use the singular, so: `consumerTestSpace`